recipe/meta.yaml

```diff
@@ -35,8 +37,10 @@ requirements:
     - click-plugins >=1.0
     - six >=1.7
     - munch
-    - shapely
+    - argparse  # [py<27]
+    - ordereddict  # [py<27]
```

Comment:
I don't think it is needed to add those (it's not that conda-forge is building py26 packages, luckily ;))

Reply:
Needed? No, and that is why I never added them before. However, for the sake of completeness of the recipe, and who knows maybe someone wants to build this locally for 2.6, it is nice to have.